tests/functional/tlsprofile/test_csi_snapshot_metadata_tlsprofile.py

```
@@ -0,0 +1,225 @@
+    The CSI Snapshot Metadata service must pick up the ODF TLS API without
+    manual restarts. Skips on FIPS (PQ / ChaCha in our cipher lists). Deletes
+    the CR at the end—only run where that is safe. An autouse fixture also
+    deletes a leftover ``ocs-tls-profile`` if the test aborts early. CSI
+    provisioner / snapshot-metadata pods may roll when TLS settings change.
+    """
+
+    @pytest.fixture(autouse=True)
+    def cleanup_tlsprofile(self, request):
+        tls = TLSProfile()
+
+        def _cleanup():
```

Ruff (0.16.2)

[warning] Missing return type annotation for private function `_cleanup`

Add return type annotation: `None`

(ANN202)